src/css/custom.css

```diff
@@ -210,6 +211,75 @@ a.btn.navbar__github::before {
   padding: 0 var(--ifm-pre-padding);
 }
 
+/* Navbar "More from SUSE" items. Thanks to Nunix. https://github.com/rancher/elemental-docs/pull/235 */
+
+.navbar__icon {
+  font-family: poppins,sans-serif;
+  font-size: 16px;
+}
+
+.navbar__icon:before {
+  content: "";
+  display: inline-flex;
+  height: 20px;
+  width: 20px;
+  margin-right: 4px;
+  background-color: var(--ifm-navbar-link-color);
+}
+
+.navbar__rancher:before {
+  mask: url(/static/img/header/icon-rancher.png) no-repeat 100% 100%;
+  mask-size: cover;
+  width: 35px;
+  padding-bottom: 7px;
```

Comment:
```suggestion
  mask-size: contain;
  width: 35px;
```

Using `contain`:

![header-contain](https://github.com/rancher/rancher-docs/assets/1832069/afb93197-68c0-46bf-8072-f19e697b5441)